ctrl/ctrl.go

```diff
@@ -175,13 +176,8 @@ func (ctrl *Ctrl) Run(ctx context.Context) error {
 				return fmt.Errorf("post-execution model map adding link: %v", err)
 			}
 
-			b, err := json.MarshalIndent(m, "", "  ")
-			if err != nil {
-				log.Error("post-execution marshalling map", "error", err)
-				return fmt.Errorf("post-execution %q marshalling map: %v", execution.Name, err)
-			}
+			outputs[execution.Name] = m
```

Comment:
Can we then add a check in the config file validation to ensure the execution names have no conflict?